communitiesuk/funding-service-design-frontend
# Fs 3581 update round definition to cater optional or mandatory application feedback survey questions

https://dluhcdigital.atlassian.net/browse/FS-3581
https://dluhcdigital.atlassian.net/browse/FS-3582


### Description
- Make feedback survey questions to optional on frontend when the config is set
- Add the optional text when the survey is optional to fill



### How to test
- set the value `is_feedback_survey_optional` to True in fund-store db and complete an application, observe that application can be submitted without the survey

### Screenshots of UI changes (if applicable)
1) `is_feedback_survey_optional` = False (Requires survey to be filled to submit the application)
![Picture1](https://github.com/communitiesuk/funding-service-design-frontend/assets/127315890/828d2107-47ba-4b82-9991-3ed3ac8a6107)
2) `is_feedback_survey_optional` = True  (Survey  is optional, application can be submitted without completing the survey)
![Picture2](https://github.com/communitiesuk/funding-service-design-frontend/assets/127315890/68a2ade8-68bc-4f7a-a402-36388e6b4c6e)
